### PR TITLE
Bug 1486940 Update openshift_logging_fluentd_hosts example

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1486940

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -379,7 +379,11 @@ xref:../install/configuring_inventory_file.adoc#configuring-node-host-labels[nod
 starting up, using this may cause a delay in ES receiving current log records.
 
 |`openshift_logging_fluentd_hosts`
-|List of nodes that should be labeled for Fluentd to be deployed.
+|List of nodes that should be labeled for Fluentd to be deployed. The default is
+to label all nodes, for example ['--all']. The null value is
```
Comment (bfallonf) [inline install_config/aggregate_logging.adoc]: So, the default is that all nodes are labeled as if you were using the --all option? Or the default option is to label all of them by using the --all option?

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -379,7 +379,11 @@ xref:../install/configuring_inventory_file.adoc#configuring-node-host-labels[nod
...
+`openshift_logging_fluentd_hosts={}`.
+To spin up Fluentd pods update the daemonset's nodeSelector to a valid label. For
```
Comment (bfallonf) [inline install_config/aggregate_logging.adoc]: I think nodeSelector would need backticks is said like this. Otherwise: s/nodeSelector/node selector

**Comment by bfallonf** [comment]
@bmcelvee A few nits :goat:

**Comment by bmcelvee** [comment]
Thanks, @bfallonf! :sheep:

**Pull request merged**